src/chatty/Chatty.java

```diff
@@ -235,10 +241,21 @@ public static String getUserDataDirectory() {
         if (settingsDir != null) {
             return settingsDir + File.separator;
         }
-        String dir = System.getProperty("user.home")
-                + File.separator 
-                + ".chatty" 
-                + File.separator;
+        String dir;
+        if (System.getenv("XDG_CONFIG_DIR") != null) {
+            dir = System.getenv("XDG_CONFIG_DIR")
```

Comment:
This should be `XDG_CONFIG_HOME`. (There is no `XDG_CONFIG_DIR` in [the specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) and `XDG_CONFIG_DIRS` is not really appropriate here, though ideally that should also be searched… but that’s a whole different subject to tackle than what this PR sets out to do.)
```suggestion
        if (System.getenv("XDG_CONFIG_HOME") != null) {
            dir = System.getenv("XDG_CONFIG_HOME")
```